### ISSUE TITLE
Login/Register/Password forms

### ISSUE THREAD

**Issue by lucyWMDE** [codingdavinci/relaunch2018#216]
**All**
- please enable language selection via the language options top right

**Login**
http://codingdavinci.snater.com/de/user/login
- remove "Neues Benutzerkonto Erstellen" link bottom left
- shouldn't "Passwort zurücksetzen" link bottom right read "Passwort Vergessen?"? is that an option in drupal (I get the impression these are default texts...).

**Passwort**
http://codingdavinci.snater.com/de/user/password
- remove "Neues Benutzerkonto Erstellen" link bottom left
- remove "Passwort zurücksetzen" link bottom right

**Register**
http://codingdavinci.snater.com/de/user/register to register project teams
http://codingdavinci.snater.com/de/user/register?role=institution to register institutions
- remove "Neues Benutzerkonto Erstellen" link bottom left
- remove "Passwort zurücksetzen" link bottom right
- remove text "Daten Ihrer Kulturinstitution"
- Is it possible to add some text to differentiate between the two registration forms? eg edit page title to "Neues Benutzerkonto erstellen: Project Team" and "Neues Benutzerkonto erstellen: Institution"

**Comment by lucyWMDE** [comment]
please also enable language selection for http://codingdavinci.snater.com/de/user/reset and any other forms/user-registration pages if possible.

**Comment by Snater** [comment]
The page titles are already translatable per interface translation: Current strings are "Create new institution account" and "Create new participant account". If those shall be different, please create a new issue.

What is http://codingdavinci.snater.com/de/user/reset?

**Comment by lucyWMDE** [comment]
Sorry to reopen, but I don't think it makes sense to make a new issue.


> The page titles are already translatable per interface translation

That's not the issue. The issue is that the controls top right of the screen are not enabled - it's not possible to switch between different translations of the page without manually editing the url. Please enable. 


> Current strings are "Create new institution account" and "Create new participant account". If those shall be different, please create a new issue.

Cool - those titles would be great... but I see now both pages have the "Create new institution account" title. The "Create new participant account" title has been overwritten. I don't think I can edit it myself.


> What is http://codingdavinci.snater.com/de/user/reset?

This is the page you get to after you request to reset your password. You are directed here when you click the reset link from the automated email.

**Comment by Snater** [comment]
1. Hm. I enabled the controls, but I noticed the change was not reflected on staging. I cleared the caches there once more and it seems to work for me now.
2. Not the case for me, might also just be related to caching.
3. We cannot do that. When toggling the language on the reset page, it is a new request to the one-time  login which, in that moment, would be expired. I just checked: The link sent out in the mail contains the language the forgot-password page has been accessed with. The user should receive the reset link in her/his interface language.
